geode-core/src/main/java/org/apache/geode/management/internal/configuration/realizers/RegionConfigRealizer.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.geode.management.internal.configuration.realizers;
+
+import org.apache.geode.cache.Cache;
+import org.apache.geode.cache.DataPolicy;
+import org.apache.geode.cache.RegionFactory;
+import org.apache.geode.cache.configuration.RegionAttributesDataPolicy;
+import org.apache.geode.cache.configuration.RegionAttributesType;
+import org.apache.geode.cache.configuration.RegionConfig;
+import org.apache.geode.internal.cache.PartitionAttributesImpl;
+
+public class RegionConfigRealizer implements ConfigurationRealizer<RegionConfig> {
```

Comment:
Could we have a `PartitionRegionConfigRealizer` and `ReplicateRegionConfigRealizer` and `NormalRegionConfigRealizer` and `ReplicateProxyRegionConfigRealizer`.... etc.. Maybe pre-optimization?

Reply:
Pre-optimization, especially since we don't have disparate ReplicateRegionConfig and PartitionedRegionConfig classes.